src/profiler/profiler.cpp

```diff
@@ -112,7 +112,7 @@ void applyHacks(HANDLE process_handle, CONTEXT32 &context)
 
 	// First, skip over any stub functions (a useless push/mov/pop header, followed by a jump).
 	// Move instead to the jump target.
-	if (ReadProcessMemory(process_handle, (LPCVOID)context.Eip, tmp, 16, &numRead) && numRead >= 16)
+	if (ReadProcessMemory(process_handle, reinterpret_cast<LPCVOID>((uintptr_t)context.Eip), tmp, 16, &numRead) && numRead >= 16)

```

Comment:
What is the purpose of this change? Does it address some compiler warning? Ideally such changes should be on their own commit and the reason for them explained in the commit message.

Reply:
It creates warnings in newer compilers (e.g. VS 2017).

Reply:
Does `(LPCVOID)(uintptr_t)context.Eip` work just as well?

Reply:
Alright. Personally I think `reinterpret_cast` makes perfect sense for types for which different kinds of conversions can be conceived (such as numeric/floating point or bytes to structs), but converting integer to pointer is fairly unambiguous, so a C cast would be OK here and have less syntactic overhead.